foundations/html_css/css-foundations.md

```diff
@@ -235,7 +236,7 @@ p {
 
 Take a quick look at [CSS Legal Color Values](https://www.w3schools.com/cssref/css_colors_legal.asp) to see how you can adjust the opacity of these colors by adding an alpha value.
 
-#### Typography Basics and `text-align`
+#### Typography Basics and text-align
```

Comment:
```suggestion
#### Typography Basics and Text-Align
```

Reply:
Hey, I believe I fixed these issues. May I resolve the conversation?